core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/WorkspaceApiService.scala

```diff
@@ -255,4 +255,11 @@ trait WorkspaceApiService extends UserInfoDirectives {
         }
       }
   }
+
+  private def workspaceDeleteMessage(maybeGoogleBucket: Option[String]): String = {
+    maybeGoogleBucket match {
+      case Some(bucketName) => s"Your Google bucket $bucketName will be deleted within 24h."
+      case None => "Your workspace has been deleted."
```

Comment:
Why isn't this covered by your test?

Reply:
The test I added tests `WorkspaceService` so this isn't covered. I'll add an API unit test to cover this